src/components/Map.vue

```diff
@@ -149,16 +155,29 @@ export default {
           buildTileUrlTemplate: this.buildTileUrlTemplate,
           crossOrigin: this.crossOriginMedia
         };
+
+        if (this.stac.type === 'Collection' && data.type === 'FeatureCollection') {
```

Comment:
Actually, I think we can also enable this for Catalogs. There's usually no map for Catalogs because there are no temporal information available, but I think it would also be nice to have this visualization for Catalogs whenever there are Items available.

Reply:
I've added something for this but couldn't find an example where it actually worked

Reply:
This is a candidate:
https://radiantearth.github.io/stac-browser/#/external/www.planet.com/data/stac/open-skysat-data/sp-crater/catalog.json